libraries/AdaptiveExpressions/ExpressionFunctions.cs

```diff
@@ -2954,15 +2954,15 @@ private static IEnumerable<object> Flatten(IEnumerable<object> list, int dept)
                         object result = null;
                         if (args.Count == 2 && !args[1].IsInteger())
                         {
-                            error = $"The second {args[1]} parameter must be an integer.";
+                            error = $"The second parameter ${args[1]} must be an integer.";
```

Comment:
> ${ [](start = 58, length = 3)

I don't think this is right. did you mean to use LG syntax in the middle of C# code?

Reply:
This is a typo. I will fix it

Reply:
fixed